pkg/vulnerabilityreport/scanner.go

```diff
@@ -29,9 +29,9 @@ type Scanner struct {
 	scheme    *runtime.Scheme
 	clientset kubernetes.Interface
 	opts      kube.ScannerOpts
+	plugin    Plugin
 	pods      *pod.Manager
 	ext.IDGenerator
-	Plugin
```

Comment:
Any reason why it's better to name it? Purely subjective, but I do think implicit encapsulation helps reduce boilerplate like `s.plugin.XXX`. Thoughts?

Reply:
I just want to quickly identify that this call / functionality is contributed by a plugin and distinguish it from the generic code. I also don't want to export the Plugin field.

Reply:
But in general I prefer shorter syntax